### ISSUE TITLE
Log Viewer: Add measurement cursor and zoom

### ISSUE THREAD

**Issue by patman13** [FrSkyRC/ETHOS-Feedback-Community#3349]
The log viewer is a very helpful feature.

I would like to have the following functionality to make it a real helpful tool:

- Add measurement cursor
- add zoom
- add find min and max values

**Comment by hrenz** [comment]
Same as #3337

**Comment by bsongis-frsky** [comment]
Yes there is a zoom!

**Comment by robertobrazill** [comment]
> Yes there is a zoom!

No, it is not possible to zoom. It is not possible to navigate the graph, that is, we cannot check all the information because the cusos does not move.
